server/api/resolvers.ts

```diff
@@ -1,31 +1,57 @@
-import { requestGroupMutationResolvers, requestGroupQueryResolvers, requestGroupResolvers } from './resolvers/requestGroupResolvers'
-import { requestMutationResolvers, requestQueryResolvers, requestResolvers } from './resolvers/requestResolvers'
-import { requestTypeMutationResolvers, requestTypeQueryResolvers, requestTypeResolvers } from './resolvers/requestTypeResolvers'
+import {
+  donationFormMutationResolvers,
+  donationFormQueryResolvers,
+  DonationItemCondition,
+  DonationItemStatus,
+} from "./resolvers/donationFormResolvers";
+
+import {
+  requestGroupMutationResolvers,
+  requestGroupQueryResolvers,
+  requestGroupResolvers,
+} from "./resolvers/requestGroupResolvers";
+
+import {
+  requestMutationResolvers,
+  requestQueryResolvers,
+  requestResolvers,
+} from "./resolvers/requestResolvers";
+
+import {
+  requestTypeMutationResolvers,
+  requestTypeQueryResolvers,
+  requestTypeResolvers,
+} from "./resolvers/requestTypeResolvers";
 
 const resolvers = {
-    Query: {
-        ...requestQueryResolvers,
-        ...requestTypeQueryResolvers,
-        ...requestGroupQueryResolvers,
-    },
-    
-    Mutation: {
-        ...requestMutationResolvers,
-        ...requestTypeMutationResolvers,
-        ...requestGroupMutationResolvers,
-    },
-
-    Request: {
-        ...requestResolvers
-    },
-
-    RequestType: {
-        ...requestTypeResolvers
-    },
-
-    RequestGroup: {
-        ...requestGroupResolvers
-    }
-}
-
-export { resolvers }
+  DonationItemCondition,
+  DonationItemStatus,
+
+  Query: {
+    ...donationFormQueryResolvers,
+    ...requestQueryResolvers,
+    ...requestTypeQueryResolvers,
+    ...requestGroupQueryResolvers,
+  },
+
+  Mutation: {
+    ...donationFormMutationResolvers,
+    ...requestMutationResolvers,
+    ...requestTypeMutationResolvers,
+    ...requestGroupMutationResolvers,
+  },
+
+  Request: {
+    ...requestResolvers,
+  },
+
+  RequestType: {
+    ...requestTypeResolvers,
+  },
+
+  RequestGroup: {
+    ...requestGroupResolvers,
+  },
```

Comment:
Need to write resolver for `requestGroup` field on `DonationForm` and add the resolver here
```
DonationForm: {
  ...donationFormResolvers
}
```

Reply:
for this resolver would we need to check that `donationForm.requestGroup` field exists and if so, should we return null?

Reply:
No need to return null.
GraphQL treats javascript `undefined` and `null` as `null`
So if `donationForm.requestGroup` does not exist, GraphQL will translate the `undefined` into a `null` (lol i think)